app/assets/stylesheets/index.scss

```diff
@@ -0,0 +1,67 @@
+.container2 {
+  width: 100%;
+  height: 100%;
+  background-color: rgb(107, 106, 106);
+  display: grid;
+  grid-column-gap: 25px;
+  grid-template-columns: repeat(6, 1fr);
+  grid-template-rows: 0.25fr repeat(3, 1fr);
+  grid-template-areas:
```

Comment:
@cortinabryan 
Like here.. This doesn't really seem very [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself). Instead, for this generic grid structure, just use `grid-template-columns`.

You only wanna name cells (defined by `grid-template-areas`) if you want to be really specific about placement; and cannot do this just with the HTML structure (like, totally regular order).